northstar/src/npk/manifest.rs

```diff
@@ -319,6 +319,13 @@ pub struct Seccomp {
     pub allow: Option<HashMap<NonNullString, SyscallRule>>,
 }
 
+impl ToString for Seccomp {
+    fn to_string(&self) -> String {
+        // unwrap(): Can only if serialize fails which it does not for the types we use.
+        serde_yaml::to_string(self).unwrap()
+    }
+}
+
```

Comment:
This feels odd.

It's a `to_yaml` and not a `to_string`.  It's also only used at a single place where even the potential error could be returned.

Reply:
Agreed, I inlined the method and got rid of the unwrap().